datasetinsights/estimators/faster_rcnn.py

```diff
@@ -20,6 +20,8 @@
 from datasetinsights.evaluation_metrics.base import EvaluationMetric
 from datasetinsights.io.bbox import BBox2D
 from datasetinsights.io.summarywriter import get_summary_writer
+from datasetinsights.io.tracker.factory import TrackerFactory
+from datasetinsights.io.tracker.mzflow import MLFlowTracker
```

Comment:
Why is this import required if the TrackerFactory can provide an instance of MLflow tracking object?

Reply:
MLFlowTracker : This import requires to refresh the token.
TrackerFactory: Requires to get tracker instance.

Reply:
Related to `refresh_token` discussion below.